src/Sulu/Component/Rest/ListBuilder/AbstractFieldDescriptor.php

```diff
@@ -208,10 +191,52 @@ public function getMinWidth()
     }
 
     /**
-     * @return string
+     * {@inheritdoc}
      */
     public function getClass()
     {
         return $this->class;
     }
+
+    /**
+     * Return array of properties.
+     *
+     * @return array
+     */
+    protected function toArray()
```

Comment:
We can remove unnecessary `to|fromArray` as `serialize` will take care of everything. These are used in JMS metadata so as not to serialize the ReflectionClass I believe.